pre-push.sh

```diff
@@ -0,0 +1,13 @@
+git stash -q --keep-index
+
+
+# Tests
+npm test
+
+RESULT=$?
+
+git stash pop -q
```

Comment:
Removed the hook implementation. We'll use travic to test builds which will defeat the purpose of pre-tests.